frontend/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,35 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule, Router } from '@angular/router';
+import { DeviceDetectorService } from 'src/app/shared/serivces/device-detector/device-detector.service';
+
```

Comment:
I have two versions of the frontend - mobile and desktop - and I implemented them as separated lazy-loading modules.